test/test_core_aten_ops.py

```diff
@@ -0,0 +1,6883 @@
+import torch_xla
+import torch_xla.core.xla_model as xm
+from torch_xla.stablehlo import exported_program_to_stablehlo
+import torch
+import torch._export
+
+import tempfile
+import unittest
+
+
+class AtenOpTest(unittest.TestCase):
+
+  def test_aten_abs_0(self):
+    args = (torch.randn((10, 10)).to(torch.float32),)
+    kwargs = dict()
+    res = torch.ops.aten.abs(*args, **kwargs)
+    exported = torch.export.export(torch.ops.aten.abs, args, kwargs)
```

Comment:
question.. what happened when we try to export the op that we do not have lowering for?

Reply:
exported_program_to_stablehlo throws exception.

Reply:
ok so I guess this test only includes ops that we have lowering? The purpose is to make sure there is no regression?

Reply:
IIRC, this test file includes all core aten ops (even unlowered ops in ptxla). Those that are unlowered are skipped with `@unittest.skip`.

Reply:
But looking at it again, I had a qq, @qihqi -- seems like many op tests here are skipped even for already lowered ops. For example, I see that `test_aten_any_0` is skipped, but we should actually have `any` lowered already -- https://github.com/pytorch/xla/blob/master/codegen/xla_native_functions.yaml#L13. Is there another reason why we're skipping certain op tests?

Reply:
Oh I see, some tests might have problems with the test code itself. Looks like this is happening on the CI, probably just need to skip those as well.

Reply:
Yes we mark skip for now the ones that dont have lowrings (or gives incorrect result). We can they file issues for those and instructions to fix